grpc-native/src/main/java/org/ballerinalang/net/grpc/Message.java

```diff
@@ -136,14 +136,15 @@ public Message(
         if (input == null) {
             if (bBMap != null) {
                 for (Map.Entry<Integer, Descriptors.FieldDescriptor> entry : fieldDescriptors.entrySet()) {
-                    BString bFieldName = io.ballerina.runtime.api.StringUtils.fromString(entry.getValue().getName());
+                    BString bFieldName =
+                            io.ballerina.runtime.api.utils.StringUtils.fromString(entry.getValue().getName());
```

Comment:
```suggestion
                            StringUtils.fromString(entry.getValue().getName());
```

Reply:
We can remove the full qualified path of `StringUtils` as we have a single import, right?